modules/cli/src/main/java/org/apache/ignite/internal/cli/core/repl/executor/ReplExecutorProvider.java

```diff
@@ -35,7 +35,7 @@ public class ReplExecutorProvider {
     private Terminal terminal;
 
     @Inject
-    private NodeNameRegistry nodeNameRegistry;
+    private NodeNameRegistryImpl nodeNameRegistry;
```

Comment:
There are multiple places where you've renamed the `NodeNameRegistry` to `NodeNameRegistryImpl` I think this is done by mistake. Could you fix all of them?

Reply:
done